app/models/lti_configuration.rb

```diff
@@ -3,6 +3,19 @@
 class LtiConfiguration < ApplicationRecord
   belongs_to :organization
 
+  enum lms_type: {
+    canvas: "Canvas",
+    blackboard: "Blackboard",
+    brightspace: "Brightspace",
+    moodle: "Moodle",
+    other: "other"
+  }, _prefix: true
+
+  def lms_name(default_name: "Other Learning Management System")
```

Comment:
I've gone ahead and added a very simple `lms_name` helper method. Essentially it will return the name corresponding the LMS platform type _unless_ that type is `:other`. In that case, it will either return a default or let us provide it depending on the view context at the time. Also added some simple tests for it.

Reply:
Love it!

Reply:
If the value of `:other` in our enum was "Other Learning Management System", wouldn't that kill the need for this method?

Reply:
@d12 Sometimes (see PR #2056) we want a different text representation for `:other` than “Other Learning Management System” on the UI — like “your Learning Management System” or “a learning management system”, etc.